src/Settings/PreferencesWindow.cpp

```diff
@@ -500,11 +501,12 @@ QTreeWidgetItem *PreferencesWindow::getTopLevelItem(const QString &text) const
     return nullptr;
 }
 
-QTreeWidgetItem *PreferencesWindow::getChild(QTreeWidgetItem *item, const QString &text) const
+QTreeWidgetItem *PreferencesWindow::getChild(QTreeWidgetItem *item, // NOLINT: It can be static
```

Comment:
This method can be made static, but since I have no idea about these codes, it is upto you to make them static and remove that `//NOLINT` suppress.